WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -216,4 +216,13 @@ def export_public
   def countries
     render json: Country.all
   end
+
+  def competition_series
+    competition_series = CompetitionSeries.find_by_wcif_id(params[:id])
```

Comment:
You can also raise the `WcaException::NotFound` if the series was actually not found. That will also avoid the null-safe `&` call below.

Reply:
Changed to a `.blank? ||` condition. Not sure if `.present?`, `.nil?`, or `.empty?` (or something else) would be more appropriate - please let me know.

Reply:
I think `if !competition_series.present?` is the cleanest approach here